numpy/_core/code_generators/generate_ufunc_api.py

```diff
@@ -36,7 +36,7 @@
   PyObject *numpy = PyImport_ImportModule("numpy._core._multiarray_umath");
   if (numpy == NULL && PyErr_ExceptionMatches(PyExc_ModuleNotFoundError)) {
     PyErr_Clear();
-    numpy = PyImport_ImportModule("numpy.core._multiarray_umath");
+    numpy = PyImport_ImportModule("numpy._core._multiarray_umath");
```

Comment:
Thanks!  Yes, we forgot this one.  This needs a try/except (effectively), if it fails, we need to clear the error and try the old path.  The `generate_numpy_api.py` should have the pattern.
```suggestion
    numpy = PyImport_ImportModule("numpy._core._multiarray_umath");
    if (numpy == NULL && PyErr_ExceptionMatches(PyExc_ModuleNotFoundError)) {
      PyErr_Clear();
      numpy = PyImport_ImportModule("numpy.core._multiarray_umath");
    }
```
This needs a try/except, because we need to support running on NumPy 1.x (I effectively copied that over from the numpy_api, so should be fine).